cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -0,0 +1,590 @@
+            pytest.fail('User {} was not found in details table after SSA run'.format(username))
+
+
+@pytest.mark.long_running
+def test_ssa_groups(ssa_vm, appliance, ssa_profile):
+    """ Tests SSA fetches correct results for groups
+
+    Metadata:
+        test_flag: vm_analysis
+    """
+    group = fauxfactory.gen_alphanumeric()
+    expected_group = None
+
+    if ssa_vm.system_type != WINDOWS:
+        # Add a new group
+        # force ssh re-connection
+        ssa_vm.ssh.close()
+        ssa_vm.ssh.run_command("groupdel {0} || groupadd {0}".format(group))
+        expected_group = ssa_vm.ssh.run_command("cat /etc/group | wc -l").output.strip('\n')
+
+    ssa_vm.smartstate_scan(wait_for_task_result=True)
+
+    # Check that all data has been fetched
+    current_group = ssa_vm.get_detail(properties=('Security', 'Groups'))
+    if ssa_vm.system_type != WINDOWS:
+        assert current_group == expected_group
+
+    # Make sure created group is in the list
+    details_property_view = ssa_vm.open_details(("Security", "Groups"))
+    if ssa_vm.system_type != WINDOWS:
+        try:
+            details_property_view.paginator.find_row_on_pages(
+                details_property_view.table, name=group)
+        except NoSuchElementException:
+            pytest.fail('Group {} was not found in details table after SSA run'.format(group))
+
+
+@pytest.mark.long_running
+def test_ssa_packages(ssa_vm, soft_assert, appliance, ssa_profile):
+    """ Tests SSA fetches correct results for packages
+
+    Metadata:
+        test_flag: vm_analysis
+    """
+
+    if ssa_vm.system_type == WINDOWS:
+        pytest.skip("Windows has no packages")
+
+    if 'package' not in ssa_vm.system_type.keys():
+        pytest.skip("Don't know how to update packages for {}".format(ssa_vm.system_type))
+
+    package_name = ssa_vm.system_type['package']
+    package_command = ssa_vm.system_type['install-command']
+    package_number_command = ssa_vm.system_type['package-number']
+
+    cmd = package_command.format(package_name)
+    # force ssh re-connection
+    ssa_vm.ssh.close()
+    output = ssa_vm.ssh.run_command(cmd.format(package_name)).output
+    logger.info("%s output:\n%s", cmd, output)
+
+    expected = ssa_vm.ssh.run_command(package_number_command).output.strip('\n')
+
+    ssa_vm.smartstate_scan(wait_for_task_result=True)
+
+    # Check that all data has been fetched
+    current = ssa_vm.get_detail(properties=('Configuration', 'Packages'))
+    assert current == expected
+
+    # Make sure new package is listed
+    details_property_view = ssa_vm.open_details(("Configuration", "Packages"))
+    try:
+        details_property_view.paginator.find_row_on_pages(
+            details_property_view.table, name=package_name)
+    except NoSuchElementException:
+        pytest.fail('Package {} was not found in details table after SSA run'.format(package_name))
+
+
+@pytest.mark.uncollectif(BZ('1528419', forced_streams=['5.8', '5.9']).blocks,
```

Comment:
**Optional**

Please use `@pytest.mark.meta(blockers=[BZ(1528419)])` so that its recorded in artifactor accurately as a BZ block, and not as an uncollected test.  You can still use forced_streams when creating the BZ object.